raster/r.out.vrml/r.out.vrml.html

```diff
@@ -58,9 +58,5 @@ <h2>TODO</h2>
 </ul>
 
 <h2>AUTHOR</h2>
-  Bill Brown, US Army Construction Engineering Research Laboratory
 
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
+Bill Brown, US Army Construction Engineering Research Laboratory
```

Comment:
`<h2>BUGS:</h2>` header should be `<h2>BUGS</h2>` without `:`.

Reply:
fixed